style.css

```diff
@@ -0,0 +1,71 @@
+html
+{
+    font-size: 10px;
+}
+body
+{
```

Comment:
please jump a line after a css declaration to keep your code clean

Reply:
> please jump a line after a css declaration to keep your code clean

Nice Job explaining the reason for jumping a line.